### PR TITLE
New logging

### PR DESCRIPTION
**Pull request by sclassen**

### CODE REVIEW

File: core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java [hunk 1]
```diff
@@ -73,6 +74,11 @@
  */
 public class CommandLine {
 
+    static {
+        // this is placed here above the any thing else to ensure no logger has been created prior to this line
+        FileLog.setLogFileNamePostfix("settings");
```
Comment (hendrikebbers) [inline core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java:79]: Why does the CommandLine has the settings prefix?

File: core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/ConsoleOutputPaneModel.java [hunk 2]
```diff
@@ -164,8 +164,8 @@ String createLine(MessageWithHeader m) {
         if (showHeaders) {
             sb.append(m.getHeader().toString(showUser,
                     showOrigin,
-                    showLevel,
                     showDate,
+                    showLevel,
                     showCode,
                     showThread1,
```
Comment (hendrikebbers) [inline core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/ConsoleOutputPaneModel.java:170]: What is the difference between thread1 and thread2 ?

File: core/src/main/java/net/sourceforge/jnlp/util/logging/FileLog.java [hunk 3]
```diff
@@ -50,26 +51,48 @@
...
+            logFileNamePrefix = envPrefix;
+        } else {
+            logFileNamePrefix = FILE_LOG_NAME_FORMATTER.format(new Date());
+        }
+
+        final String envPostfix = System.getenv(LOG_POSTFIX_ENV);
+        if (!StringUtils.isBlank(envPostfix)) {
+            logFileNamePostfix = envPostfix;
+        } else {
+            logFileNamePostfix = "itw";
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/logging/FileLog.java:76]: we already log in an itw folder. Do we really need as a prefix, too?

File: core/src/main/java/net/sourceforge/jnlp/util/logging/OutputControllerLevel.java [hunk 4]
```diff
@@ -12,11 +12,17 @@
 
     ;
 
+    private final String displayName = (name() + "   ").substring(0,5);
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/logging/OutputControllerLevel.java:15]: Can you do this in the constructor?

File: core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java (hunk 1 above)
Comment (sclassen) [reply core/src/main/java/net/adoptopenjdk/icedteaweb/client/commandline/CommandLine.java:79]: the `main()` in CommandLine is used by ITW to start the settings UI.
This change marks the logs from running the settings by appending "settings" to the log file name

File: core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/ConsoleOutputPaneModel.java (hunk 2 above)
Comment (sclassen) [reply core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/ConsoleOutputPaneModel.java:170]: one is the hash and other the name of the thread.
I renamed the vars in most locations. will also do it here

File: core/src/main/java/net/sourceforge/jnlp/util/logging/FileLog.java (hunk 3 above)
Comment (sclassen) [reply core/src/main/java/net/sourceforge/jnlp/util/logging/FileLog.java:76]: what else would you do as a postfix???

File: core/src/main/java/net/sourceforge/jnlp/util/logging/OutputControllerLevel.java (hunk 4 above)
Comment (sclassen) [reply core/src/main/java/net/sourceforge/jnlp/util/logging/OutputControllerLevel.java:15]: what would be the difference???

**Comment by sclassen** [comment]
I merge this for now.
Feel free to improve

**Pull request merged**